_topic_maps/_topic_map.yml

```diff
@@ -290,6 +290,10 @@ Topics:
   Topics:
   - Name: Installing an on-premise cluster using the Assisted Installer
```

Comment:
not related to changes in this PR, but I see:
``http://jowilkin.com:8080/TELCODOCS-754/installing/installing_on_prem_assisted/installing-on-prem-assisted.html#pre-installation-considerations_installing-one-prem-assisted``
(notice the ``one`` in the URL)